app/assets/javascripts/discourse/app/templates/components/user-card-contents.hbs

```diff
@@ -62,6 +62,9 @@
           {{#if this.user.staged}}
             <h2 class="staged">{{i18n "user.staged"}}</h2>
           {{/if}}
+          {{#if siteSettings.enable_user_status}}
+            <h3 class="user-status">{{html-safe this.userStatus}}</h3>
```

Comment:
You have a security issue here, nothing stops a user to have malicious code in `userStatus`, I could for example have this css as my status:

```
<style>body {display: none;}</style>
```

And anyone looking at my user card would have a blank screen, there are most likely way more dangerous attacks than this possible :)

Reply:
That was a huge flaw! Thank you for catching this and for the explanation!